yarn.lock

```diff
@@ -8921,6 +8921,7 @@ __metadata:
     react-router-dom: ^5.2.1
     react-scripts: 5.0.0
     react-table: ^7.7.0
+    react-tooltip: ^4.2.21
```

Comment:
This component has a nice look. If we want to use it though, we should update *all* tooltips in the application to use it as well. 

Currently I think the following components have tooltips (look for the attribute `title=`)
- InlineLabel
- EventLog
- ForwarderTooltip
- QuantifySlider